spinbot/policy/stale_issue_policy.py

```diff
@@ -7,6 +7,7 @@ def __init__(self):
         super().__init__()
         self.stale_days = self.config.get('stale_days')
         self.ignore_lifecycle_label = self.config.get('ignore_lifecycle_label', 'no-lifecycle')
+        self.beginner_friendly_label = self.config.get('beginner-friendly_label', 'beginner-friendly')
```

Comment:
Not sure if anyone has context on this, but I was curious why we're attempting to read these label values from config rather than just using a string constant? Is this so Spinbot could potentially be configured to other open-source projects with different label conventions?